dlk/python/dlk/templates/src/func/impl/apply_thresholds.cpp

```diff
@@ -54,6 +54,8 @@ void ApplyThresholds(
           new_d = 2;
         else
           new_d = 3;
+      } else if (flag == 0) { // ignore
+        new_d = 0;
```

Comment:
Why do you need this one? :thinking:

Reply:
threshold data is only provided "original" number of channels, and rest of data are filled with zero.
In this process, rest of data should be ignored (new_d can be arbitrary number between 0 and 3).

Reply:
Uh, new_d must be 0 (if next op is conv).

Reply:
Hmm, I see.

BTW, new_d could be anything nut keeping it zero, may simplify debug... :thinking: 

If the next operator is conv, the number of output channels is already multiple of 32, i.e. there is no padding on kernels/thresholds. as the number of input, channels has to be 32. Otherwise, here is space2depth or something else in between, which must to ignore all "non" existent channels inside of packed words.

Reply:
> If the next operator is conv, the number of output channels is already multiple of 32
Oh, really?

Reply:
Yep, as so far a number of input channels must be multiple of 32.
Ok, there are some exceptions, like Lookup which produce less than 32 and pads rest with zeroes.
But in all other places, we were always relying on input channels to be a power of 32.

Reply:
OK, what new_d should be?